src/components/Error/index.ts

```diff
@@ -0,0 +1 @@
+export * from './Error';
```

Comment:
It's recommended to use an explicit name when re-exporting components from a module. This helps eliminate assumptions about the call stack of any errors. 📝